### DISCUSSION TITLE
Playback won't work if overlays are enabled.

### DISCUSSION THREAD

**Discussion by moa12gg** [gopro/labs#424] [Q&A] [answered]
I also tried factory reset.
You are using the latest firmware.
It's been 3 days since I bought the GoPro 11.

my shooting environment
1080p 60 fps
wide fov
autoboost
10bit off

We do not use other qr codes other than overlays.
If you use the overlay command below
Sometimes it freezes during recording, or when playing back after recording, the green screen or screen freezes like a picture, or the camera freezes.

**g0!MBRNO=10!MBURN="(0,0)[BRyyyy-mm-dd HH:MM:SS aa 'GPS53%1.2f'm/s]"**

Please forgive the awkwardness of using a translator.
Thank you in advance for your reply.

![KakaoTalk_20230628_141114416_01](https://github.com/gopro/labs/assets/92944493/c53259bb-1f9a-4454-88db-4f2a20934586)
![KakaoTalk_20230628_141114416](https://github.com/gopro/labs/assets/92944493/61896d98-f1e8-4e5b-9fb7-ee558a757803)

**Comment by moa12gg** [comment]
The self answer is:
If you adjust the size of the overlay, the above phenomenon occurs.
But i need a fixed size.

Default:
Overlay vertical size 20
Overlay horizontal size 0
Offset from the edge 10
Limit display time

The default is not work for me Can you fix this?

**Comment by dnewman-gpsw** [comment, marked as answer]
The default is working only, really smaller is the only way to improve stability.  This is one of the more stressful hacks for the camera. If you want large fonts, you need to print less information.  e.g. this worked for me g0!MBRNO=10!MBURN="(0,40)[BRHH:MM:SS]"
